tests/ui/specs/DayPicker/DayPicker-GetDayStinrg-specs.js

```diff
@@ -0,0 +1,175 @@
+const Page = require('./DayPickerPage');
+
+describe('DayPicker', function () {
+	beforeEach(async function () {
+		await Page.open('GetDayStinrg');
```

Comment:
Please fix the typo. Also file name need to be fixed.

Reply:
Suggested changes.